### PR TITLE
Fluff Paper Language Setting

### PR DESCRIPTION
**Pull request by RustingWithYou**
Fluff papers can now have a written language set, as the code for translating things was copypasted in like 6 places.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Comment by FluffyGhoster** [comment]
Merge master to fix the CI

**Review by NonQueueingMatt — APPROVED** [review]

File: maps/away/ships/unathi_pirate/hiskyn/unathi_pirate_hiskyn.dm [hunk 1]
```diff
@@ -141,10 +141,4 @@
 /obj/item/paper/fluff/hiskyn
 	name = "KEEP OUT!"
 	info = "How many times I gotta warn you tailwags about going through a man's stash! Next time I'm putting a landmine in this damned thing!"
-
-/obj/item/paper/fluff/hiskyn/Initialize()
-	. = ..()
-	var/languagetext = "\[lang=o\]"
-	languagetext += "[info]\[/lang\]"
-	info = parsepencode(languagetext)
-	icon_state = "paper_words"
```
Comment (NonQueueingMatt) [inline maps/away/ships/unathi_pirate/hiskyn/unathi_pirate_hiskyn.dm:150]: You removed the custom icon state on a lot of these, not sure if intentional

File: maps/random_ruins/exoplanets/haneunim/haneunim_crash.dm [hunk 2]
```diff
@@ -75,11 +75,4 @@
...
 	info = "They fucking sold us out. I told the LT not to make a deal with those 5-Cheung fucks, but did she listen? No, course not. Bastards are barely better than the corps. Fucking KASF were on us before we had time to blink. Blew out half the engines, was barely able to get us down here, but\
 	it'd take a miracle to get back up again. Either gonna wait for the Konyangers to come arrest us, or wait to starve. If anyone finds this, I didn't want to be a smuggler. Just saw what the corps were doing to Sol, figured I had to take a stand. Fat lot of good that did. Whoever's reading this, take my body back to Mars. Bury me at home, if there's anything left of it."
-
-
-/obj/item/paper/fluff/haneunim_crash/Initialize()
-	. = ..()
-	var/languagetext = "\[lang=1\]"
-	languagetext += "[info]\[/lang\]"
-	info = parsepencode(languagetext)
-	icon_state = "scrap_bloodied"
```
Comment (NonQueueingMatt) [inline maps/random_ruins/exoplanets/haneunim/haneunim_crash.dm:85]: You removed the custom icon state on a lot of these, not sure if intentional

File: maps/random_ruins/exoplanets/haneunim/haneunim_refugees.dm [hunk 3]
```diff
@@ -31,10 +31,4 @@
 	name = "message"
 	desc = "A few words, scratched onto the back of a Go-Go-Gwok receipt."
 	info = "Remember us, who strived for freedom and failed. Remember us, upon Ascension."
-
-/obj/item/paper/fluff/haneunim_refugees/Initialize()
-	. = ..()
-	var/languagetext = "\[lang=6]"
-	languagetext += "[info]\[/lang\]"
-	info = parsepencode(languagetext)
-	icon_state = "paper_words"
```
Comment (NonQueueingMatt) [inline maps/random_ruins/exoplanets/haneunim/haneunim_refugees.dm:40]: You removed the custom icon state on a lot of these, not sure if intentional

File: maps/away/ships/coc/coc_scarab/coc_scarab.dm [hunk 4]
```diff
@@ -203,13 +203,7 @@
...
-
-/obj/item/paper/fluff/scarabengine/Initialize()
-	. = ..()
-	var/languagetext = "\[lang=3\]"
-	languagetext += "[info]\[/lang\]"
-	info = parsepencode(languagetext)
-	icon_state = "paper_words"
```
Comment (NonQueueingMatt) [inline maps/away/ships/coc/coc_scarab/coc_scarab.dm:212]: You removed the custom icon state on a lot of these, not sure if intentional. Paper_words should probably be set automatically if there's a language?

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: maps/away/ships/coc/coc_scarab/coc_scarab.dm (hunk 4 above)
Comment (RustingWithYou) [reply maps/away/ships/coc/coc_scarab/coc_scarab.dm:212]: Calling update_icon will set the icon state correctly if the paper has info

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

**Pull request merged**